api/src/main/java/org/pmiops/workbench/actionaudit/targetproperties/ProfileTargetProperty.kt

```diff
@@ -0,0 +1,24 @@
+package org.pmiops.workbench.actionaudit.targetproperties
+
+import org.pmiops.workbench.model.Profile
+
+// N.B. entries will rarely be referred to by name, but are accessed via values(). I.e. they are
+// not safe to remove, even if an IDE indicates otherwise.
+enum class ProfileTargetProperty
```

Comment:
Are these to create immutable versions? I am not entirely clear on the purpose? Just wanted to check on my understanding of these.